kibbeh/src/modules/user/UserPage.tsx

```diff
@@ -11,6 +12,11 @@ export const UserPage: PageComponent<UserPageProps> = ({}) => {
   const { query } = useRouter();
   return (
     <DefaultDesktopLayout>
+      <HeaderController
+        title={
+          query.username && !Array.isArray(query.username) ? query.username : ""
```

Comment:
why are you checking if the username is an array?

Reply:
> why are you checking if the username is not an array?

The type of the query.username is string | string[] | undefined, so I added it just to make sure I don't get the type string[].